tests/integration_tests/test_wave_01_02_03.py

```diff
@@ -1,7 +1,6 @@
 import pytest
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
-
 @pytest.mark.skip
```

Comment:
You forgot to unskip your integration tests. But hey, they pass, so that's good news!